CONTRIBUTING.md

```diff
@@ -21,3 +21,25 @@ The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build u
 Retarget the solution to v141:
 1. Right click the ReactNative solution and click "Retarget Solution"
 2. Make sure the Platform Toolset has Upgrade to v141 selected, and click "OK."
+
+
+## Using RNTester
+RNTester is a React Native Windows App that demonstrates the implemented views and modules of React Native Windows.
+
+You can use it to test your changes to react-native-windows by making sure your changes haven't broken the views and modules.
+
+#### But How?
+Before starting make sure you have run `npm install` in the react-native-windows directory. Additionally, make sure the RNTester submodule is up to date by running `git pull --recursive-submodules` from the react-native-windows directory.
+
+Use Visual Studio 2015 or higher, with the Windows 10 SDK 10.0.10586 or higher.
```

Comment:
Should this be 10.0.14393 at this point?

Reply:
I honestly don't know, I got that from the FB RNTester readme. You're probably right, I'll change it

Reply:
FB RNTester README is wrong as well. Everything should be at 10.0.14393 at this point.